mmv1/templates/terraform/operation.go.erb

```diff
@@ -23,7 +23,11 @@ func (w *<%= product_name -%>OperationWaiter) QueryOp() (interface{}, error) {
     return nil, fmt.Errorf("Cannot query operation, it's unset or nil.")
   }
   // Returns the proper get.
+  <% if async.operation&.full_url -%>
+  url := fmt.Sprintf("<%= async.operation.full_url.gsub('{{op_id}}', '%s') -%>", w.CommonOperationWaiter.Op.Name)
```

Comment:
So each request has to go to the localized endpoint, but any operation requests go to the main one? That's kind of a nightmare...

Reply:
sorry for delay, but I was out some days, yeah I don't know why this API was designed in that way, but is more complicated to implement when you compare with other products.